samples/agent-test-app/src/main/java/com/newrelic/agent/android/testapp/MainActivity.java

```diff
@@ -63,6 +63,8 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // Add NewRelic
         NewRelic.enableFeature(FeatureFlag.NativeReporting);
+        NewRelic.enableFeature(FeatureFlag.LogReporting);
```

Comment:
@cthomas-newrelic enable feature flag here

Reply:
Done in incoming PR.